### DISCUSSION TITLE
HDD issue

### DISCUSSION THREAD

**Discussion by Ibrahim-ElSayed** [rogerfar/rdt-client#551] [General]
**What version are you using?**
2.0.8
**What OS are you running?**
Windows 11
**Are you using Docker or as a service?**
Service
**Which debrid provider are you using?**
Real Debrid
**Which downloader are you using?**
Internal
**Please attach a log file here with the log setting set to debug**

I'm not sure if this is just a me thing, but when I download to my HDD (which is slow) at 20mb/s, my monitors would turn black then turn back on, then my pc would blue screen. With my SSD, there was no issue.
Just wanted to point out that issue.

**Comment by rogerfar** [comment]
I don't think you're really using 2.0.8, but sounds more like a memory issue on your end, try using another downloader and seeif that helps.
